server/src/main/java/org/opensearch/index/engine/RamAccountingRefreshListener.java

```diff
@@ -1,94 +0,0 @@
-/*
- * SPDX-License-Identifier: Apache-2.0
- *
- * The OpenSearch Contributors require contributions made to
- * this file be licensed under the Apache-2.0 license or a
- * compatible open source license.
- */
-
-/*
- * Licensed to Elasticsearch under one or more contributor
- * license agreements. See the NOTICE file distributed with
- * this work for additional information regarding copyright
- * ownership. Elasticsearch licenses this file to you under
- * the Apache License, Version 2.0 (the "License"); you may
- * not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *    http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing,
- * software distributed under the License is distributed on an
- * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
- * KIND, either express or implied.  See the License for the
- * specific language governing permissions and limitations
- * under the License.
- */
-
-/*
- * Modifications Copyright OpenSearch Contributors. See
- * GitHub history for details.
- */
-
-package org.opensearch.index.engine;
-
-import org.apache.lucene.index.IndexReader;
-import org.apache.lucene.index.LeafReaderContext;
-import org.apache.lucene.index.SegmentReader;
-import org.opensearch.common.breaker.CircuitBreaker;
-import org.opensearch.common.lucene.Lucene;
-import org.opensearch.common.lucene.index.OpenSearchDirectoryReader;
-import org.opensearch.indices.breaker.CircuitBreakerService;
-
-import java.util.Collections;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Set;
-import java.util.function.BiConsumer;
-
-/**
- * A refresh listener that tracks the amount of memory used by segments in the accounting circuit breaker.
- */
-final class RamAccountingRefreshListener implements BiConsumer<OpenSearchDirectoryReader, OpenSearchDirectoryReader> {
-
-    private final CircuitBreakerService breakerService;
-
-    RamAccountingRefreshListener(CircuitBreakerService breakerService) {
-        this.breakerService = breakerService;
-    }
-
-    @Override
-    public void accept(OpenSearchDirectoryReader reader, OpenSearchDirectoryReader previousReader) {
-        final CircuitBreaker breaker = breakerService.getBreaker(CircuitBreaker.ACCOUNTING);
```

Comment:
Since we removed it, I believe `CircuitBreaker.ACCOUNTING` becomes useless as well, seems like this is the only place where it was managed (please correct me if I am wrong).

Reply:
Ah, good catch. We can remove the Accountable. I'll take care of that here as well.

Reply:
On second thought. I'm going to remove that in a follow up PR to keep the changes contained.